pages/signup.tsx

```diff
@@ -111,100 +103,100 @@ export default function SignUp({
   const $graffiti = useField(FIELDS.graffiti)
   const $github = useField(FIELDS.github)
   const $country = useField(FIELDS.country)
-  const { executeRecaptcha } = useGoogleReCaptcha()
+  // const { executeRecaptcha } = useGoogleReCaptcha()
 
-  const [$error, $setError] = useState<string>(UNSET)
-  const [$signedUp, $setSignedUp] = useState<boolean>(false)
+  // const [$error, $setError] = useState<string>(UNSET)
+  // const [$signedUp, $setSignedUp] = useState<boolean>(false)
   const [$loaded, $setLoaded] = useState<boolean>(false)
   useEffect(() => {
     if ($country?.label) {
       $setLoaded(true)
     }
   }, [$setLoaded, $country])
-  const testInvalid = useCallback(() => {
-    const noEmail = !$email?.touched
-    const noGraffiti = !$graffiti?.touched
-    const noGithub = !$github?.touched
-    // for old men
-    const noCountry = !$country?.touched
-    const untouched = noEmail || noGraffiti || noCountry
-    const invalid =
-      !$email?.valid || !$graffiti?.valid || !$social?.valid || !$country?.valid
+  // const testInvalid = useCallback(() => {
```

Comment:
Is there a better way we could remove this functionality other than just commenting it all out? Looks pretty messy + there's not reasoning for when future people come to look at the code